composer.json

```diff
@@ -57,15 +57,16 @@
         "php-http/cache-plugin": "^1.7",
         "php-http/mock-client": "^1.2",
         "php-http/promise": "^1.0",
-        "symfony/browser-kit": "^5.4 || ^6.0 || ^7.0",
-        "symfony/cache": "^5.4 || ^6.0 || ^7.0",
-        "symfony/dom-crawler": "^5.4 || ^6.0 || ^7.0",
-        "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0",
-        "symfony/http-foundation": "^5.4 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^7.0.3",
-        "symfony/stopwatch": "^5.4 || ^6.0 || ^7.0",
-        "symfony/twig-bundle": "^5.4 || ^6.0 || ^7.0",
-        "symfony/web-profiler-bundle": "^5.4 || ^6.0 || ^7.0",
+        "phpunit/phpunit": "^9",
```

Comment:
when we use the phpunit-bridge, we don't need the phpunit dependency. in 1.x i just ditched the bridge because it ended up with incompatible versions, we should do that in 2.x too (but use a more recent version than 9 if we can)

Reply:
for this pull request, i suggest removing the line again. i will then cleanup the 2.x branch by merging 1.x to 2.x

Reply:
I added an explanation of why it was added in commit https://github.com/php-http/HttplugBundle/pull/463/commits/0725be77660379b018bbbe31deec26084be83a7a

Without this line - composer installs PHPUnit 10 which is incompatible with the `phpunit-bridge`.

So I can remove it only if I will stick `matthiasnoback/symfony-dependency-injection-test` for a version that is compatible only with PHPUnit 9

Reply:
ah, i see.

i will try to clean this up in a separate PR (getting rid of phpunit-bridge and upgrading phpunit if possible)